add: document SELECT.elements (#532)
cap-js/docs: node.js/cds-ql.md

### .elements {.property}

> The CSN outline of the selected elements as an object. Key is the selected element or alias , value is the CSN definition:

Reviewer: ```suggestion
The CSN outline of the selected elements as an object. Key is the selected element or alias, value is the CSN definition:
```

> A query like 

Reviewer: ```suggestion
Let's assume the following query:
```

```js
SELECT.from('sap.capire.bookshop.Books').columns('ID', 'title')
```

> is represented within `.elements` as:

Reviewer: ```suggestion
This query is represented within `.elements` as:
```

```js
{
  ID: number { key: true, type: 'cds.Integer' },
  title: string {
    '@mandatory': true,
    localized: true,
    type: 'cds.String',
    length: 111,
    '@Common.FieldControl': { '#': 'Mandatory' }
  }
}
```

> This is useful for custom implementation that acts on the selection of specific elements.

Reviewer: ```suggestion
This is useful for custom implementations that act on the selection of specific elements.
```
